Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -471,7 +471,7 @@ extension CarPlayManager {
     }
     
     internal func calculate(_ options: RouteOptions, completionHandler: @escaping Directions.RouteCompletionHandler) {
-        directions.calculateWithCache(options: options, completionHandler: completionHandler)
+        NavigationRouter().requestRoutes(options: options, completionHandler: completionHandler)
```

Comment:
I wanted to support 2 cases: where user need just 1 request, and when he needs to make several requests in a row. Depending on it, he should (not)retain the router instance. According to Native team, it is not too expensive to instantiate or deallocate such routers, so there's minimal sense to store shared instances

Reply:
Creating CacheHandle is expensive object to create, @mskurydin please confirm. 

Any reasons not to retain router in Navigator?

Reply:
Typically, router is needed once every 2 minutes for route refreshing. If this is not enabled, router will be called only at the beginning of the active guidance or when user offroutes. It seems like keeping it alive has no benefits since most of the time it does nothing.